File: Scripts/Sample/CMB01/q3_mem.java
import java.util.*;
import java.io.*;
class Pract
{
  public static void main(String []args)throws Exception
	{
		BufferedReader br=new BufferedReader(new  InputStreamReader(System.in));
		Integer N=Integer.parseInt(br.readLine());
		for(int i=0;i<N;i++)
		{	int b=0;
			int i1=0,i2=0;
			String s[]=(br.readLine()).split(" ");
			int n1=Integer.parseInt(s[0]);
			int n2=Integer.parseInt(s[1]);
			i1=Integer.parseInt(new StringBuilder(n1+"").reverse().toString());
			i2=Integer.parseInt(new StringBuilder(n2+"").reverse().toString());
			int temp=Integer.parseInt(new 

StringBuilder((i1+i2)+"").reverse().toString());
					 
			System.out.println(temp);
		}
	}
 static int Reverse(int i){
		int d,m=0;
		while(i>0)
		{
			d=i%10;
			i=i/10;
			m=m*10+d;	
		}
	 return m;
	}
} 

File: Scripts/Sample/CMB01/q3_mem.py
#by ksdthegr8

def reverse(num):
    d = 1
    f = 0
    e = 0
    g = '0'
    h = 0
    for x in range(len(str(num))):
        d = d * 10
        f = d / 10
        e = int((num % d - num % f) / f)
        g = g + str(e)
        h = int(g)
    return(h)

n = int(input())
for x in range(n):
    l = input()
    (a1,b1) = l.split(' ')
    a, b = int(a1), int(b1)
    a2,b2 = reverse(a), reverse(b)
    c = reverse(a2+b2)
    print(c)
    